client/lib/impure-lodash/index.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * External dependencies
+ */
+import {
```

Comment:
yeah it's 100% for mocking during tests, but I wasn't sure if you were implying another way of structuring it so it's less of an integral part of the components that use it.